docker/Dockerfile

```diff
@@ -34,19 +35,27 @@ ENV LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/usr/local/openmpi/lib
 
 ENV NCCL_LAUNCH_MODE=GROUP
 
+#install nccl
+RUN wget https://developer.download.nvidia.com/compute/cuda/repos/ubuntu2204/x86_64/cuda-keyring_1.0-1_all.deb && \
+    dpkg -i cuda-keyring_1.0-1_all.deb && \
+    apt update -y && \
+    apt install libnccl2 libnccl-dev
+
 # Should be in the lmdeploy root directory when building docker image
 COPY . /opt/lmdeploy
 
 WORKDIR /opt/lmdeploy
 
 RUN cd /opt/lmdeploy &&\
+    curl https://sh.rustup.rs -sSf | sh -s -- -y &&\
```

Comment:
pls move this to RUN command where installing sys packages

Reply:
i have move it now